src/styles/main.scss

```diff
@@ -0,0 +1,6 @@
+@import  "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/keyframes";
+
+@import "./blocks/watch.scss";
+@import "./blocks/page.scss";
```

Comment:
Please update the file extension from .scss to .css as the import statement should reference the compiled CSS file 🔄